test/integration/targets/iosxr_logging/tests/netconf/basic.yaml

```diff
@@ -93,6 +93,9 @@
     provider: "{{ netconf }}"
   register: result
 
+- debug:
```

Comment:
Is this debug msg required?

Reply:
Yes, this test fails here on DCI, want to have debug printed